### DISCUSSION TITLE
FIR support on some devices

### DISCUSSION THREAD

**Discussion by jmpsmash** [mrene/minidsp-rs#310] [Q&A] [answered]
Hi, this is a brilliant program!

There are some devices that doesn't expose FIR filter support, eg. SHD and DDRC88. How do they interact with the FIR support via minidsp-rs? Will it not work or will it somehow expose FIR support in these devices?

**Comment by mrene** [comment, marked as answer]
It will fail to apply them if the device doesn't support the feature. It's just replicating the commands sent by the official apps.
